network-setup.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+
+docker-compose run --rm wpcli core multisite-install \
+	--url="localhost" \
+	--base="/" \
+	--title="MIT Libraries" \
+	--admin_email="mjbernha@mit.edu" \
```

Comment:
this probably doesn't super matter, but it might be better to embed an entirely fake account like `fake@example.com` or something to make it clear it doesn't matter (if it doesn't matter)